ui/lib/core/addon/components/ttl-picker2.js

```diff
@@ -63,6 +64,10 @@ export default TtlForm.extend({
     const changeOnInit = this.changeOnInit;
     // if initial value is unset use params passed in as defaults
     if (!value && value !== 0) {
+      // make sure what is displayed is set on the model.
+      // for example if NOT using OpenAPI, you hide the Toggle and don't set a defaultValue
+      // TTL is displaying 30s but at this point you're sending nothing on the model.
+      next(() => this.handleChange());
```

Comment:
IMO I don't think we should be calling `onChange` within init. If the issue is that the model isn't set on load, that's an issue with the model's default value being set. 
We could, to prevent a value being shown that will not be the default set, set enableTTL to false if there isn't an initial value